uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -61,11 +126,18 @@ sub new {
     return bless $self, $params;
 }
 
-################################################################
-## writeErrorLog and update Notification Table##################
-## this is a useful function that will close the log and write #
-## error messages in case of abnormal program termination ######
-################################################################
+
+=pod
+
+=head3 writeErrorLog($message, $failStatus, $LogDir)
+
+Writes error log. This is a useful function that will close the log and write
+error messages in case of abnormal program termination.
+
+INPUT: notification message, fail status of the process, log directory
```

Comment:
INPUTS instead of INPUT (although this is consistent in the rest of the script, but I believe the previous ones we were using INPUTS)